egs/multi_conv_zh_es_ta/ST/hent_srt/model.py

```
@@ -691,26 +691,31 @@ def forward(
+                x = time_warp(
+                    x,
+                    time_warp_factor=time_warp_factor,
+                    supervision_segments=supervision_segments,
+                )
+                # Independently apply frequency masking and time masking to the two copies
+                x = spec_augment(x.repeat(2, 1, 1))
+            else:
+                x = x.repeat(2, 1, 1)
+            x_lens = x_lens.repeat(2)
+            y = k2.ragged.cat([y, y], axis=0)
+            if self.st_joiner != None and self.use_st_ctc:
```

Ruff (0.15.18)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)